### ISSUE TITLE
Death Crimson 2

### ISSUE THREAD

**Issue by animekai** [Rinnegatamante/flycast-vita-Compatibility#17]
# Game summary
- Game name: Death Crimson 2
- Game version: JP CDI

# flycast summary
- Version: v Pre-Release

# Issues

Game is playable but slow, arounds like 15fps to 30fps, and 60fps sometimes,

Edit: The pointer is super fast, trying to shooting things is a little difficult, not to much.
# Screenshots
![2021-01-02-110404-008560](https://user-images.githubusercontent.com/32076097/103461283-324f1080-4ceb-11eb-995f-2f4805f217a1.png)
![2021-01-02-110438-676971](https://user-images.githubusercontent.com/32076097/103461284-33803d80-4ceb-11eb-9002-4ca2ac0b1ea2.png)
![2021-01-02-110456-311837](https://user-images.githubusercontent.com/32076097/103461285-3418d400-4ceb-11eb-966f-74f2d95520c0.png)
![2021-01-02-110508-790370](https://user-images.githubusercontent.com/32076097/103461286-3418d400-4ceb-11eb-9aa5-c9b32669e4e1.png)
![2021-01-02-110600-542788](https://user-images.githubusercontent.com/32076097/103461287-34b16a80-4ceb-11eb-84b9-c762d363815f.png)
![2021-01-02-110652-302450](https://user-images.githubusercontent.com/32076097/103461288-34b16a80-4ceb-11eb-9648-df3d0da31a93.png)
![2021-01-02-110713-448573](https://user-images.githubusercontent.com/32076097/103461289-354a0100-4ceb-11eb-9ccb-a12d2c0db57c.png)
![2021-01-02-110742-694131](https://user-images.githubusercontent.com/32076097/103461290-35e29780-4ceb-11eb-9a56-0728d191e25a.png)
![2021-01-02-110830-725014](https://user-images.githubusercontent.com/32076097/103461291-35e29780-4ceb-11eb-9957-82366638aa30.png)

# Recommended labels

- A Playable
- B?
- C?

**Comment by Rinnegatamante** [comment]
Waiting for a confirmation the game is Playable to reopen this.

**Comment by animekai** [comment]
yes this game is playable, the framerate is acceptable for gameplay

**Comment by georgewaraw** [comment]
This one actually crashes for me on 0.1 as soon as I try to boot it.
